### PR TITLE
Add Goldman Sachs recap blog post

### PR DESCRIPTION
**Pull request by colebow**
Just took a screengrab of the video for the header, I think it looks ok.

### CODE REVIEW

File: _posts/2022-12-05-trino-summit-2022-goldman-sachs-recap.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Leveraging Trino to Power Data at Goldman Sachs"
```
Comment (mosabua) [inline _posts/2022-12-05-trino-summit-2022-goldman-sachs-recap.md:3]: power data

File: _posts/2022-12-05-trino-summit-2022-goldman-sachs-recap.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+author: "Sumit Halder, Siddhant Chadha, Suman-Newton, Ramesh Bhanan, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/goldman-sachs.png
+---
+
+Continuing with [the Trino Summit 2022 sessions posts]({% post_url
+2022-11-21-trino-summit-2022-recap %}), we're diving into an insightful
+lightning talk from [Goldman Sachs](https://www.goldmansachs.com) about how they
+use Trino to ensure data quality across the board for all users and customers.
+By using Trino to federate their various data sources, querying everything in
```
Comment (mosabua) [inline _posts/2022-12-05-trino-summit-2022-goldman-sachs-recap.md:13]: Thats a long sentence.. can you break it up.

File: _posts/2022-12-05-trino-summit-2022-goldman-sachs-recap.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+<!--more-->
+
+{% youtube g9fLA3tFG-Q %}
+
+## Recap
+
+Validating data quality can be a tricky and complicated process. Data resides
+in many sources, with different rules and different processes for checking
+quality, and Goldman's data ingestion team may not have a detailed understanding
```
Comment (mosabua) [inline _posts/2022-12-05-trino-summit-2022-goldman-sachs-recap.md:26]:
```suggestion
quality. Goldman's data ingestion team may not have a detailed understanding
```

File: _posts/2022-12-05-trino-summit-2022-goldman-sachs-recap.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+of all data sets. Despite that, there is a need to autonomously verify and
+validate all data to be confident in its quality and integrity. The solution to
+this challenge? A queryable data quality platform powered by Trino.
+
+The underlying data quality platform's logic handles the validation, and resting
```
Comment (mosabua) [inline _posts/2022-12-05-trino-summit-2022-goldman-sachs-recap.md:31]:
```suggestion
The underlying data quality platform's logic handles the validation. Resting
```

File: _posts/2022-12-05-trino-summit-2022-goldman-sachs-recap.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+on top of it is Trino, the scalable, fast solution to ensure that users can
+query what they need. Even when the platform is profiling the data, enforcing
+various quality rules, and validating the data in different ways, Trino is there
```
Comment (mosabua) [inline _posts/2022-12-05-trino-summit-2022-goldman-sachs-recap.md:34]:
```suggestion
various quality rules, and validating the data in different ways, Trino is there.
```

File: _posts/2022-12-05-trino-summit-2022-goldman-sachs-recap.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+to provide access to everything contained within, proving that quality, speed,
```
Comment (mosabua) [inline _posts/2022-12-05-trino-summit-2022-goldman-sachs-recap.md:35]:
```suggestion
Trino provides access to everything contained within, proving that quality, speed,
```

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2022-12-05-trino-summit-2022-goldman-sachs-recap.md (hunk 5 above)
Comment (colebow) [reply _posts/2022-12-05-trino-summit-2022-goldman-sachs-recap.md:34]: I like leaving this as-is, I think breaking it up into two sentences loses meaning, and we've shortened all the sentences before this, so we can afford to have a longer one. Writing works and flows better when sentence length varies.

**Review by mosabua — APPROVED** [review]
Time to ship it!

**Pull request merged**